#Replace include of "projects.h" with "proj_api.h" and remove use of what is
#now considered an internal structure (struct PJ). 

#Proj.4 has removed "projects.h" from the installation of that library, because
#much that's in it is considered private implementation details that should
#not have been exposed.  That many packages use those internals directly is
#considered a problem in those other packages, and a bug request in proj.4's
#bug tracker to reinstate projects.h as an installed file was closed with
#the status "wontfix".  Codes that use the public API of proj.4 are supposed
#to include "proj_api.h" and use the API calls, not direct access to internal
#data structures or functions that are not intended for use outside the
#library.
--- src/map_tif.c	2010/01/31 02:12:24	1.45
+++ src/map_tif.c	2012/05/15 15:29:36	1.46
@@ -93,7 +93,7 @@
 
 // Needed for GTIFProj4FromLatLong() replacement below
 #ifdef HAVE_LIBGDAL
-#  include "projects.h"
+#  include "proj_api.h"
 #endif  // HAVE_LIBGDAL
 
 
@@ -378,7 +378,7 @@
                              double *padfY ) {
 
     char    *pszProjection;
-    PJ      *psPJ;
+    projPJ      psPJ;
     int     i;
 
 // --------------------------------------------------------------------
